src/EFCore.PG/Metadata/NpgsqlIndexAnnotations.cs

```diff
@@ -53,5 +54,34 @@ public string Method
         protected virtual bool SetMethod(string value)
             => Annotations.SetAnnotation(NpgsqlAnnotationNames.IndexMethod, value);
 
+        /// <summary>
+        /// The PostgreSQL operators to be used with each index field.
+        /// </summary>
+        /// <remarks>
+        /// https://www.postgresql.org/docs/current/static/indexes-opclass.html
+        /// </remarks>
+        public string[] Operators
+        {
+            get => GetOperators();
+            set => SetOperators(value);
+        }
+
+        protected virtual string[] GetOperators() {
+            var value = (string)Annotations.Metadata[NpgsqlAnnotationNames.IndexOperators];
+            if (value != null) {
+                return value.Split(' ');
+            }
+            return null;
+        }
+
+        protected virtual bool SetOperators(string[] operators) {
+            if (operators != null && operators.Count > 0) {
+                Annotations.SetAnnotation(NpgsqlAnnotationNames.IndexMethod, operators.Join(' '));
```

Comment:
Since an annotation can contain string array, there shouldn't be need to do a join here. Else it is causing inconsistency in the code that is the annotation value is string or string[]